index.d.ts

```diff
@@ -0,0 +1,5 @@
+type anyObject = {
```

Comment:
Lets use an interface instead of a type, and export that too. Also rename it to `Params`
```suggestion
export interface Params  {
 ... 
}
```